### PR TITLE
[spirv] Allow single tile configurations for cooperative matrix

### PR DESCRIPTION
**Pull request by qedawkins**
Removes a check in the verifier that required at least two tiles per workgroup as the motivating case has since been fixed.

### CODE REVIEW

File: compiler/src/iree/compiler/Codegen/SPIRV/test/illegal_configuration.mlir [hunk 1]
```diff
@@ -153,43 +153,6 @@ hal.executable private @matmul_tensors {
 
 // -----
 
-#compilation = #iree_codegen.compilation_info<
-    lowering_config  = <tile_sizes = [[32, 64], [4, 2], [0, 0, 4]]>,
-    translation_info = <SPIRVMatmulPromoteVectorize>,
```
Comment (antiagainst) [inline compiler/src/iree/compiler/Codegen/SPIRV/test/illegal_configuration.mlir:158]: This test is unrelated to cooperative matrix pipeline?

Comment (qedawkins) [reply compiler/src/iree/compiler/Codegen/SPIRV/test/illegal_configuration.mlir]: oh, this shouldn't have been dropped. I added it back.

**Review by antiagainst — APPROVED** [review]
